VERSION

```diff
@@ -1 +1 @@
-99.9.9
+0.13.0
```

Comment:
Revert this change, please

Reply:
Done! I already reverted it and pushed the branch.
I'll wait to see if this change dissapears from the diff and then mark the conversation as resolved, would that be right?